src/events/event_handler.ts

```diff
@@ -127,6 +128,59 @@ export class eventHandler {
 		this.handleChanges(filePath, currentText);
 	}
 
+	handleTabChangeEvent = () => {
+		/*
+		The flow for handler will be as follows:
+			- For the current open repoPath, get the repo_id and from config File
+			- Get Current Open Tabs TabGroups.ALL
+			- For each tab:
+				* Get the Relative Path of each file using repoPath
+				* Get the fileID using relative_path found in above step
+		*/
+		// Record timestamp
+		const created_at = new Date().getTime();
+		console.log("created_at: ", created_at);
+		// For the current open repoPath, get the repo_id and from config File
+		const configUtils = new ConfigUtils();
+		const repoId = configUtils.getRepoIdByPath(this.repoPath);
+		console.log(`Repo ID: ${repoId}`)
+		const configJSON = configUtils.config;
+
+		// Get list of current tabs
+		const open_tabs = vscode.window.tabGroups.all;
+		let tabs: any[] = [];
+		// Loop through tab groups
+		for (const tab_group of open_tabs) {
+			for (let tab of tab_group.tabs) {
+				// console.log(`Displaying tabs: `, tab);
+				// Get path of tab
+				// @ts-ignore
```

Comment:
Why do we have ts-ignore here?

Reply:
As ts wasn't recognizing the paths of `tabs`:
![image](https://github.com/codesyncapp/codesync-vs/assets/15974895/a8bf903b-22d5-49a3-a1e5-583542198418)

Reply:
It still doesn't recognize after we add: `if(!tabs.input) return;`

![image](https://github.com/codesyncapp/codesync-vs/assets/15974895/35a18bf0-7cd1-47a2-819f-db92aaa0a2ed)